lib/instrumentation/undici.js

```diff
@@ -116,22 +116,33 @@ function addDTHeaders({ transaction, config, request }) {
  */
 function createExternalSegment({ shim, request, parentSegment }) {
   const url = new URL(request.origin + request.path)
-  const name = NAMES.EXTERNAL.PREFIX + url.host + url.pathname
+  const obfuscatedPath = urltils.obfuscatePath(shim.agent.config, url.pathname)
+  const name = NAMES.EXTERNAL.PREFIX + url.host + obfuscatedPath
   // Metrics for `External/<host>` will have a suffix of undici
   // We will have to see if this matters for people only using fetch
   // It's undici under the hood so ¯\_(ツ)_/¯
   const segment = shim.createSegment(name, recordExternal(url.host, 'undici'), parentSegment)
+
+  // brown-fields alert!!
```

Comment:
What does this mean? I genuinely don't know what "brown-fields" means.

Reply:
i guess it's incorrect spelling. i was trying to be funny that this code base is riddled with landmines and this is one of them

Reply:
i inserted a hyphen but was referring to [brownfield](https://en.wikipedia.org/wiki/Brownfield_(software_development))